### ISSUE TITLE
SpriteFont description files - game does not distinguish styles

### ISSUE THREAD

**Issue by energyuser** [MonoGame/MonoGame#4641]
``` xml
        <!--
            Style controls the style of the font. Valid entries are "Regular", "Bold", "Italic",
            and "Bold, Italic", and are case sensitive.
        -->
        <Style>Italic</Style>
```

I tried build/rebuild/clean in both VS and the Content pipeline app and nothing... The font remains `Regular`. I even installed the new Stable version of Monogame 3.5 and ran it again. Still not fixed. I tried deleting the XNB file and building again, a new one got created and it's `Regular`. I tried with other fonts and couldn't change their style as well.

The font above was first built with `Regular` style some months ago. Today I decided to change it to `Italic` to see what it would look like in my game and it stays `Regular`. I also had the same problem with a font that was built as `Bold` and won't change to regular...

Must be the way the app is building the font XNBs?

**Comment by Tzenchor** [comment]
That is a work in progress.
See #4537 and #4561

**Comment by energyuser** [comment]
Yes, I've been reading and looking forward to the merging of these pull requests ever since I read them. And I still keep checking for updates. But aren't these about improving the way Monogame handles fonts and bring it closer/identical to XNA? Or this also fixes the style bug?

**Comment by Tzenchor** [comment]
It's specific for styles, the first one tries to font the correct font in the font family while the second simulates the style by modifying the bitmaps

**Comment by energyuser** [comment]
But I did not suppose styling was **entirely** broken... Anyway, is there any ETA on when this is getting merged? Or at least a workaround for the issue?

**Comment by tomspilman** [comment]
@energyuser - The work around is install the italic/bold/regular style for the font you want to use.  This way you get a specific designed font and not an automatically generated one.

**Comment by energyuser** [comment]
That's just great... And what about an ETA for fixing the font bugs? I'm really looking forward to this since my entire game GUI is based on SpriteFonts.

**Comment by polyminthe** [comment]
I agree me too, i'm looking forward an issue about that problem. Do you can add that fix in the next update please ? Thank's for your great job. I really like use your library

**Comment by tomspilman** [comment]
@energyuser @polyminthe - The work is being done in https://github.com/mono/MonoGame/pull/4561 and should be merged in the next week or two I expect.

**Comment by energyuser** [comment]
When is #4561 getting merged?

**Comment by tomspilman** [comment]
@energyuser - I expect to be merging it this next week.

**Comment by energyuser** [comment]
@tomspilman Is it ready yet?

**Comment by Tzenchor** [comment]
@energyuser Give @tomspilman time, he is busy with many things. Subscribe to #4561 and you'll get notifications when it moves forward

**Comment by energyuser** [comment]
So this is going to be fixed in 3.6?

**Comment by wldevries** [comment]
Duplicate of https://github.com/MonoGame/MonoGame/issues/4149

**Comment by ThomasFOG** [comment]
Closing in favor of #4149
